src/arima.cpp

```diff
@@ -400,70 +494,86 @@ void getQ0(const py::array_t<double> phiv, const py::array_t<double> thetav,
     }
     return;
   }
+
   if (p > 0) {
     std::vector<double> rbar(nrbar);
     std::vector<double> thetab(np);
     std::vector<double> xnext(np);
     std::vector<double> xrow(np);
-    ind = 0;
-    int ind1 = -1;
-    int npr = np - r;
-    int npr1 = npr + 1;
-    int indj = npr;
-    int ind2 = npr - 1;
-    for (int j = 0; j < r; ++j) {
-      double phij = j < p ? phi[j] : 0.0;
+
+    size_t ind = 0;
+    ssize_t ind1 = -1;
```

Comment:
Seems like windows doesn't like this
```suggestion
    py::ssize_t ind1 = -1;
```

Reply:
Didn't realize it was a posix type, fixed.